.github/workflows/publish-helm-charts.yml

```
@@ -51,14 +64,17 @@ jobs:
         shell: bash
         working-directory: ./airbyte/charts
         run: |
-          sed -i "s/    version: placeholder/    version: ${{ needs.generate-semantic-version.outputs.next-version }}/g" airbyte/Chart.yaml
+          sed -i -E "s/    version: [[:digit:]].[[:digit:]].[[:digit:]]/    version: ${{ needs.generate-semantic-version.outputs.next-version }}/g" airbyte/Chart.yaml
+          sed -i -E 's/version: [0-9]+\.[0-9]+\.[0-9]+/version: ${{ needs.generate-semantic-version.outputs.next-version }}/' airbyte/Chart.yaml
+
 
       - name: "Helm package"
         shell: bash
         run: |
```

```suggestion
        run: |
            set -x # or is it set +x for actual verbose stuff, I never remember.
```
I think if anyone has to debug this, more output would help